docs/content/operator/validator-node/configs/archives.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: Archives
+description: The archive is a historical record of all transactions on IOTA. Enable archiving on your full nodes as a best practice.
```

Comment:
```suggestion
description: The archive is a historical record of all transactions on IOTA. Enable archiving on your validator nodes as a best practice.
```

Is it even best practice on validator nodes?

Reply:
Fixed in ede15b84a110e29ed56b96fd6ceade6336c84587